### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sweatybridge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,13 +130,13 @@ If you have requests for supporting a specific flavour, please file an issue on
 
 ## Available Images
 
-### flask-gunicorn
+### [flask-gunicorn](https://hub.docker.com/repository/docker/basisai/express-flask)
```
Comment (nglinh) [inline README.md:133]:
```suggestion
### [express-flask](https://hub.docker.com/repository/docker/basisai/express-flask)
```

File: README.md [hunk 2]
```diff
@@ -130,13 +130,13 @@ If you have requests for supporting a specific flavour, please file an issue on
...
 
 Standard model server image using flask-gunicorn stack. Requires
 
 - Python 3.6+
 
-### fastapi-gunicorn
+### [fastapi-gunicorn](https://hub.docker.com/repository/docker/basisai/express-fastapi)
```
Comment (nglinh) [inline README.md:139]:
```suggestion
### [express-fastapi](https://hub.docker.com/repository/docker/basisai/express-fastapi)
```

File: README.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ serve {
 }
 ```
 
-The [basisai/express-flask](https://hub.docker.com/r/basisai/express-flask) image is hosted on docker hub and should be accessible from your workload environment on Bedrock.
+The [basisai/express-flask](https://hub.docker.com/repository/docker/basisai/express-flask) image is hosted on docker hub and should be accessible from your workload environment on Bedrock. You may change it to any other supported image, such as [basisai/express-fastapi](https://hub.docker.com/repository/docker/basisai/express-fastapi).
```
Comment (nglinh) [inline README.md:66]: Can we use the old link? This link requires logging in docker hub for some reason

**Review by nglinh — APPROVED** [review]
Let's refer to the same tag as dockerhub to minimize confusion

File: README.md (hunk 1 above)
Comment (sweatybridge) [reply README.md:133]: hmm it's named after the directory in this repo though

File: README.md (hunk 3 above)
Comment (sweatybridge) [reply README.md:66]: oic, yea let me change it back

File: README.md (hunk 1 above)
Comment (nglinh) [reply README.md:133]: I'm fine either way but let's use 1 only. It's easier to find an less confusion.

Comment (sweatybridge) [reply README.md:133]: Let's stick with express-flask then. The directory structure may change if we support cpp models in the future.

**Pull request merged**